mapper/tedge_mapper/src/main.rs

```diff
@@ -1,14 +1,35 @@
+use crate::error::MapperError;
+use crate::time_provider::SystemTimeProvider;
+use flockfile::{Flockfile, FlockfileError};
 use mqtt_client::Client;
+use std::path::PathBuf;
+use tedge_config::{
+    AzureMapperTimestamp, ConfigRepository, ConfigSettingAccessor, TEdgeConfigRepository,
+};
 use tracing::{debug_span, error, info, Instrument};
 
+mod az_mapper;
+mod c8y_mapper;
+mod error;
 mod mapper;
+mod time_provider;
 
-const APP_NAME: &str = "tedge-mapper";
-const DEFAULT_LOG_LEVEL: &str = "warn";
+const APP_NAME_C8Y: &str = "tedge-mapper-c8y";
+const APP_NAME_AZ: &str = "tedge-mapper-az";
+const DEFAULT_LOG_LEVEL: &str = "debug";
```

Comment:
I don't mind changing the default log level, but is there any reason why we want to use `debug` for releases? I suggest go no lower than `info`.
```suggestion
const DEFAULT_LOG_LEVEL: &str = "info";
```

Reply:
Oh yeah, good catch. It's my mistake when I was testing it. Forgot to put it back to `info`. Thanks!

Reply:
For testing you can still use RUST_LOG variable (we most likely will drop support for it in the future, but for now you can use it):
```shell
RUST_LOG=debug ./target/debug/tedge_mapper 
```

Reply:
By the way, it was `warn`. So, I'll put it back to `warn`.

Reply:
True, but `info` seems reasonable as well.

Reply:
I also like `info` because it shows `tedge-mapper starts!` by `info`. I'll ask Sebastian when it gets UX review.